sdk/ai/azure-ai-projects/samples/agents/sample_agents_code_interpreter_attachment_enterprise_search.py

```diff
@@ -62,7 +60,12 @@
     ds = VectorStoreDataSource(asset_identifier=asset_uri, asset_type=VectorStoreDataSourceAssetType.URI_ASSET)
 
     # create a message with the attachment
-    attachment = MessageAttachment(data_sources=[ds], tools=code_interpreter.definitions)
+    attachment = MessageAttachment(
+        data_source=ds,
+        tools=[
+                FileSearchTool().definitions[0],
```

Comment:
We should not need to define both file search and code interpreter

Reply:
Fixed.